include/tool-myank.h

```diff
@@ -175,6 +180,9 @@ class Tool_myank : public HumTool {
 		vector<MeasureInfo> MeasureInList;  // used with -m option
 		vector<vector<MyCoord> > metstates;
 
+		string linesQ;
```

Comment:
I reserve `Q` to mean a boolean (Q = question).

Myank does not use `m_` prefix as well.  If you want add them you can; otherwise, I will update the member variables to do so some time in the future.